src/types.jl

```diff
@@ -190,7 +190,7 @@ isdiv(x)  = isa_SymType(Val(:Div), x)
 Base.isequal(::Symbolic, x) = false
 Base.isequal(x, ::Symbolic) = false
 Base.isequal(::Symbolic, ::Symbolic) = false
-
+coeff_isequal(a, b) = isequal(a, b) || ((a isa AbstractFloat && b isa AbstractFloat) && (a==b))
```

Comment:
```suggestion
coeff_isequal(a, b) = isequal(a, b) || ((a isa AbstractFloat || b isa AbstractFloat) && (a==b))
```
From the commit https://github.com/JuliaSymbolics/SymbolicUtils.jl/commit/972e675519c5a6a35763e6c657e49f73c0d3115a message, this seems to be used for dealing with the comparison of `Float64` `-0.0` and `Int` `0`. 
So maybe the condition here should be changed from both being floating-point to either one being floating-point?

Reply:
Yeah, this makes sense.